Danki2/Assets/Scripts/Utils/GameObjectExtensions.cs

```diff
@@ -2,10 +2,8 @@
 
 public static class GameObjectExtensions
 {
-    public static bool TryGetComponent<T>(this GameObject @object, out T component)
-        where T : MonoBehaviour
+    public static void SetLayer(this GameObject gameObject, Layer layer)
     {
-        component = @object.GetComponent<T>();
-        return component != null;
```

Comment:
I've used this method in the mortal cache in another branch, so we should keep it around (even though it was my idea to get rid of it...)

Reply:
Haha ok no problem